packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/ExchangeForm/CurrencySelect/CurrencySelect.js

```diff
@@ -77,4 +78,4 @@ const mapDispatchToProps = dispatch => ({
   actions: bindActionCreators(actions.components.exchange, dispatch)
 })
 
-export default connect(getData, mapDispatchToProps)(CurrencySelect)
+export default connect(getData, mapDispatchToProps)(memo(CurrencySelect))
```

Comment:
Why is this necessary?

Reply:
I was testing something, ignore this, will revert. 
We have constant call of render of CurrencySelect on swap page, have to do some more digging why it occur like that